stdlib/src/commonMain/kotlin/it/unibo/collektive/stdlib/Gossip.kt

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2024, Danilo Pianini, Nicolas Farabegoli, Elisa Tronetti,
+ * and all authors listed in the `build.gradle.kts` and the generated `pom.xml` file.
+ *
+ * This file is part of Collektive, and is distributed under the terms of the Apache License 2.0,
+ * as described in the LICENSE file in this project's repository's top directory.
+ */
+
+package it.unibo.collektive.stdlib
+
+import it.unibo.collektive.aggregate.api.Aggregate
+import it.unibo.collektive.aggregate.api.operators.share
+import it.unibo.collektive.field.Field.Companion.fold
+import it.unibo.collektive.field.Field.Companion.foldWithId
+import it.unibo.collektive.field.Field.Companion.hood
+
+/**
+ * A collection of self-stabilizing gossip algorithms.
+ */
+object SelfStabilizingGossip {
+    /**
+     * Self-stabilizing gossip-max.
+     * Spreads across all (aligned) devices the current maximum [Value] of [local],
+     * as computed by [comparator].
+     */
+    fun <ID : Comparable<ID>, Value> Aggregate<ID>.gossipMax(
+        local: Value,
+        comparator: Comparator<Value>,
+    ): Value {
```

Comment:
Add:
```suggestion
    fun <ID : Comparable<ID>, Value: Comparable<Value>> Aggregate<ID>.gossipMax(local: Value): Value =
        gossipMax(local) { a, b -> a.compareTo(b) }
```

and similar for `gossipMin`

I am also convinced that we should not use `ID`s. We should probably also take a UUIDGenerator function in the most general implementation